sbt-plugin/src/main/scala/ch/epfl/scala/SubmitDependencyGraph.scala

```diff
@@ -96,12 +96,23 @@ object SubmitDependencyGraph {
       snapshot <- getSnapshot(httpResp)
     } yield {
       state.log.info(s"Submitted successfully as $snapshotUrl/${snapshot.id}")
+      setGithubOutputs(
+        "submission-id" -> s"${snapshot.id}",
+        "submission-api-url" -> s"${snapshotUrl}/${snapshot.id}"
+      )
```

Comment:
One thought I had here was to write `snapshotJson` to a temp file and expose path to that file as an action output. That would allow downstream integration working with the JSON directly, instead of making an unnecessary call to the API. WDYT?

Reply:
@adpi2 what do you think about the idea of writing snapshot JSON to a file and providing the path as an output?

Reply:
If the JSON file is written to a Github runner, how could we access it?

Reply:
The file on the runner will be available to the next step in the workflow. It can be used directly, or user can add https://github.com/actions/upload-artifact step to make the file available to other jobs or for manual download.

Reply:
Here are some potential usage example:

```yaml
jobs:
  dependency-graph:
    name: Update Dependency Graph
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: scalacenter/sbt-dependency-submission@v2
        id: dependency-submission

      - name: Check submitted snapshot
        run: gh api '${{ steps.dependency-submission.outpus.submission-api-url }}' | jq

      - name: Upload snapshot JSON
        uses: actions/upload-artifact@v3
        with:
          name: dependency-snapshot.json
          path: ${{ steps.dependency-submission.outpus.snapshot-json-path }}
```

Reply:
Thanks for the clarification. I like the idea.

Reply:
Great! I added it in https://github.com/scalacenter/sbt-dependency-submission/pull/115/files/0c1e45b3275c2325a8575bc7977bc9e35ccbae5e..d3c391cfd4247489971ee2955ed71d5ac03a251d